packages/server-side-render/src/utils.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import { pickBy, isEmpty, mapValues } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { getBlockSupport } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { ATTRIBUTE_PROPERTY, STYLE_PROPERTY } from './constants';
+
+const identity = ( x ) => x;
+
+/**
+ * Check whether serialization of specific block support feature or set should
+ * be skipped.
+ *
+ * @param {string} blockType  Block name.
+ * @param {string} featureSet Name of block support feature set.
+ * @param {string} feature    Name of the individual feature to check.
+ *
+ * @return {boolean} Whether serialization should occur.
+ */
+function shouldSkipSerialization( blockType, featureSet, feature ) {
```

Comment:
This function is the same as: 
https://github.com/WordPress/gutenberg/blob/f26adefac0c5c3c52eed6689e5b8113dd11ebf51/packages/block-editor/src/hooks/utils.js#L101